skimage/exposure/exposure.py

```diff
@@ -505,3 +505,49 @@ def is_low_contrast(image, fraction_threshold=0.05, lower_percentile=1,
     ratio = (limits[1] - limits[0]) / (dlimits[1] - dlimits[0])
 
     return ratio < fraction_threshold
+
+
+def hist_map(old_hist, new_hist):
+    """
+    Build a function that maps from one distribution onto another.
+
+    Parameters
+    ----------
+    old_hist : tuple
+        Histogram arrays ``(count, bins)`` like those from ``numpy.histogram``,
+        where bins give edges, such that ``len(bins) == 1 + len(counts)``.
```

Comment:
why choose the ``numpy`` convention (``len(bins) == 1 + len(counts)``) and not the one of ``skimage.exposure.histogram`` (``len(bins) == len(counts)``) ?